HHS/Head-Start-TTADP
# [TTAHUB-1896] Scrape job clean up and listeners cleanup

## Description of change

This is from @GarrettEHill PR: https://github.com/HHS/Head-Start-TTADP/pull/1640

The idea here is that with the rate limiter in place (https://github.com/HHS/Head-Start-TTADP/pull/1750) we wont need any special handling of queue exceptions.

This includes includes the handling of various resource types and increase the listeners.

## How to test

- Run the resource scrape job for a few moths ensure it updates both titles, meta data, and mime types (new column).
- Add a new resource from the UI make sure the above is also updated (both PDF and URL).
- The worker should show no errors for valid resource URL's.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1896


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
